Error: Building package forest:x64-windows/x86-windows failed with: BUILD_FAILED

I tried to install coroutine with command `.\vcpkg.exe install forest:x64-windows`. I got this error:

> The following packages will be built and installed:
    forest[core]:x64-windows
Starting package 1/1: forest:x64-windows
Building package forest[core]:x64-windows...
-- Downloading https://github.com/xorz57/forest/archive/10.1.1.tar.gz...
-- Downloading https://github.com/xorz57/forest/archive/10.1.1.tar.gz... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:156 (message):
/
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues
/
Call Stack (most recent call first):
  scripts/cmake/vcpkg_from_github.cmake:108 (vcpkg_download_distfile)
  ports/forest/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:71 (include)
/
/
Error: Building package forest:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: forest:x64-windows
  Vcpkg version: 2018.11.23-nohash
/
Additionally, attach any relevant sections from the log files above.

I also tried to install forest: x86-windows with command `.\vcpkg.exe install forest:x86-windows`. I got the same error.

--- a/ports/forest/CONTROL
+++ b/ports/forest/CONTROL
@@ -1,3 +1,3 @@
 Source: forest
-Version: 10.1.1
+Version: 10.1.1-1
 Description: Template library of tree data structures

--- a/ports/forest/portfile.cmake
+++ b/ports/forest/portfile.cmake
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO xorz57/forest
-    REF 10.1.1
-    SHA512 145f60bc202c7a0ab0dad514e08606ace45780855f71d9ba6a9461c5242b751a3fefa25101387f797ecc53198cd792859927b6bd2c2aa4a1f114a311d0b74e26
+    REF 10.1.7
+    SHA512 104cf76817faf7f6cf82b8c5168b7e611a29de13cf948c55bea02ae97909756d726a8130380a6bc8c96b86466eb779a178f6ffceac16cbc24af468eb19e7dcc9
     HEAD_REF master
 )
 

